packages/uicore/src/components/MultiSelectDropDown/MultiSelectDropDown.tsx

```diff
@@ -155,9 +157,10 @@ export function MultiSelectDropDown(props: MultiSelectDropDownProps): React.Reac
           style={width ? { width } : { minWidth }}
           className={cx(
             css.dropdownButton,
-            { [css.withBorder]: !isLabel },
-            { [css.selected]: selectedItems.length > 0 },
-            { [css.minWidth]: !width }
+            { [css.withBorder]: !isLabel && showBaseInputBlue },
+            { [css.selected]: selectedItems.length > 0 && !disabled && showBaseInputBlue },
+            { [css.minWidth]: !width },
+            { [css.disabledBg]: disabled }
```

Comment:
We are already adding a 'disabled' class on line 152. Why again?

Reply:
Resolved